client/js/options.js

```diff
@@ -280,9 +280,9 @@ function initialize() {
 		$warningUnsupported.show();
 	}
 
-	$settings.on("change", "input, select, textarea", function(e) {
-		// We only want to trigger on human triggerd changes.
-		if (e.originalEvent) {
+	$settings.on("change", "input, select, textarea", function(e, {force} = {}) {
+		// We only want to trigger on human triggered changes.
+		if (e.originalEvent || force) {
```

Comment:
I don't think check is needed anymore as options no longer trigger a fake `change`. Right @creesch?

Reply:
Not entirely, synced settings will still be applied to their respective input fields which might trigger a fake change. So it probably needs to stay just for that.

Reply:
However, force is not needed at all as far as I can tell. @astorije what are you attempting to do here? This seems like moving away again from the settings refactoring I did, you should use `processSetting()` if you want to update a setting.

Reply:
Done!